Documentation/configuration.md

```diff
@@ -270,3 +270,38 @@ and password `gle`.
 Note that _within_ a particular configuration directory (either system or
 local), it is a syntax error for the same Docker registry to be defined in
 multiple files.
+
+
+
+### rktKind: `directories`
+
+This kind of configuration is used to customize the various directories that
+rkt uses. The configuration files should be placed inside a `directories`
+subdirectory (e.g. in `/usr/lib/rkt/directories` or
+`/etc/rkt/directories`).
+
+#### rktVersion: `v1`
+
+##### Description and examples
+
+The configuration works by mapping types of directories to their path in the
```

Comment:
I would prefer the docs to be kept in similar style as the other docs here. So first paragraph like "This version of `KIND` configuration specifies N additional fields: `FOO`, `BAR`, ..., `BAZ` and `QUUX`". And following paragraphs would describe each field. If there is something more to say about this kind then describe it in following paragraph. After that - an example.